gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -229,9 +229,10 @@ public LeaseAttemptStatus tryAcquireLease(DagActionStore.DagAction flowAction, l
     // Query lease arbiter table about this flow action
     Optional<GetEventInfoResult> getResult = getExistingEventInfo(flowAction, isReminderEvent);
 
+    // TODO: change all the `CASE N: ...` statements back to debug statements after uncovering issue
     try {
       if (!getResult.isPresent()) {
-        log.debug("tryAcquireLease for [{}, is; {}, eventTimestamp: {}] - CASE 1: no existing row for this flow action,"
```

Comment:
JFYI, if you wanted to see all debug statements for a specific class, you could have done this in a log4j properties file. 

That would be the easier (and faster change) since you can make it on your own internal project quickly

Reply:
agreed